src/meta/test/meta_http_service_test.cpp

```diff
@@ -300,5 +309,49 @@ TEST_F(meta_bulk_load_http_test, query_bulk_load_request)
     drop_app(NOT_BULK_LOAD);
 }
 
+TEST_F(meta_bulk_load_http_test, start_compaction_test)
+{
+    struct start_compaction_test
+    {
+        std::string request_json;
+        http_status_code expected_code;
+        std::string expected_response_json;
+    } tests[] = {
+        {"{\"app_name\":\"test_bulk_load\",\"type\":\"once\",\"target_level\":-1,\"bottommost_"
```

Comment:
Better to use [string_literal](https://en.cppreference.com/w/cpp/language/string_literal) to simplify code.